ast/parser.go

```diff
@@ -131,6 +159,15 @@ func (p *Parser) Parse() ([]Statement, []*Comment, Errors) {
 		if rules := p.parseRules(); rules != nil {
 			for i := range rules {
 				stmts = append(stmts, rules[i])
+				// Append schema annotation to rule if there is one
+				ruleLoc := rules[i].Location.Row
+				annot, err := p.getAnnotation(ruleLoc - 1)
```

Comment:
I think we should make the annotation parsing optional/conditional. In other words, the caller that creates the parser can set a flag on the parser to indicate they want to have annotations processed. Inside the eval subcommand we can enable the flag when the files are loaded. This will mitigate the impact on performance when the caller doesn't care about annotations.

Reply:
Done 860ab29